site/source/tutorials/1040_dojo1_dijits/index.md

```diff
@@ -0,0 +1,191 @@
+
+{% instruction 'Create a `/src/dojo1.cfg.js` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dojo1.cfg.js' %}
+
+{% task 'Ensure the application builds.' %}
+
+At this point, you should be able to build your application.
+
+{% instruction 'Run `dojo build` in the project root.' %}
+
+{% aside 'Reminder' %}
+You can now use `dojo build -w` to continue to develop, with live rebuilding of your project.
+{% endaside %}
+
+You should see a rather long output indicating many dependencies have been output.
+
+{% section %}
+
+## Working with Dijits
+
+To integrate Dijits within into a Dojo 2 application, they need to _wrapped_ for management by the Dojo 2 widgeting system.  As discussed in previous steps, the `DijitWrapper` function will _convert_ a Dojo 1 Dijit constructor function into something that behaves like a Dojo 2 widget class.  When used with the Dojo 1 typings, you will also get code intellisense when configuring your Dijits.
+
+The `DijitWrapper` function accepts two arguments.  The first is the Dojo 1 `declare` class.  The second is an optional string, which represents the tag name that should be used when the Dojo 2 widgeting system needs to create a _DOM stub_ to attach the Dijit instance to.  It defaults to `div` if not supplied.
+
+{% task 'Wrap Dijits.' %}
+
+{% aside 'Information' %}
+All Dojo 1 modules are AMD modules without default exports.  As such, you need to import Dojo 1 modules using the `import * as Dijit from 'module/id';` syntax.
+{% endaside %}
+
+We recommend any wrapped Dijits be created as individual modules in `/src/dijit` in the same path structure they are located in the source `dijit` package.
+
+{% instruction 'Create `/src/dijit/Fieldset.ts` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dijit/Fieldset.ts' %}
+
+{% instruction 'Create `/src/dijit/form/Button.ts` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dijit/form/Button.ts' %}
+
+{% instruction 'Create `/src/dijit/form/TextBox.ts` with the following content:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/dijit/form/TextBox.ts' %}
+
+{% task 'Import Dijits.' %}
+
+Now that you have wrapped Dijits for use, we need to import them into their containing Dojo 2 or modules.
+
+{% instruction 'Import Dijits into `/src/widgets/WorkerForm.ts`:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:5-7 %}
+
+{% section %}
+
+## Placing Dijits
+
+{% task 'Add Dijits to a widget\'s render function.' %}
+
+Now that we have imported our wrapped Dijits into the module they will be used, we need to add them to the render function.
+
+{% instruction 'Add imported Dijits to the `WorkerForm` widget\'s render function:' %}
+
+{% include_codefile 'demo/finished/biz-e-corp/src/widgets/WorkerForm.ts' lines:44-74 %}
```

Comment:
I'm getting the following errors after I drop this in:

```
ERROR in ./src/widgets/WorkerForm.ts
(52,16): error TS2345: Argument of type '{ key: string; placeHolder: string; onInput: ({ target: { value: firstName } }: TypedTargetEvent<...' is not assignable to parameter of type 'Readonly<Partial<TextBox & DeclareCreatedObject> & DijitWrapperProperties> & Readonly<WidgetPrope...'.
  Type '{ key: string; placeHolder: string; onInput: ({ target: { value: firstName } }: TypedTargetEvent<...' is not assignable to type 'Readonly<Partial<TextBox & DeclareCreatedObject> & DijitWrapperProperties>'.
    Types of property 'onInput' are incompatible.
      Type '({ target: { value: firstName } }: TypedTargetEvent<HTMLInputElement>) => void' is not assignable to type '((e: DocumentEvent) => void) | undefined'.
        Type '({ target: { value: firstName } }: TypedTargetEvent<HTMLInputElement>) => void' is not assignable to type '(e: DocumentEvent) => void'.
          Types of parameters '__0' and 'e' are incompatible.
            Type 'DocumentEvent' is not assignable to type 'TypedTargetEvent<HTMLInputElement>'.
              Property 'target' is missing in type 'DocumentEvent'.

ERROR in ./src/widgets/WorkerForm.ts
(57,16): error TS2345: Argument of type '{ key: string; placeHolder: string; onInput: ({ target: { value: lastName } }: TypedTargetEvent<H...' is not assignable to parameter of type 'Readonly<Partial<TextBox & DeclareCreatedObject> & DijitWrapperProperties> & Readonly<WidgetPrope...'.
  Type '{ key: string; placeHolder: string; onInput: ({ target: { value: lastName } }: TypedTargetEvent<H...' is not assignable to type 'Readonly<Partial<TextBox & DeclareCreatedObject> & DijitWrapperProperties>'.
    Types of property 'onInput' are incompatible.
      Type '({ target: { value: lastName } }: TypedTargetEvent<HTMLInputElement>) => void' is not assignable to type '((e: DocumentEvent) =>void) | undefined'.
        Type '({ target: { value: lastName } }: TypedTargetEvent<HTMLInputElement>) => void' is not assignable to type '(e: DocumentEvent) => void'.
          Types of parameters '__0' and 'e' are incompatible.
            Type 'DocumentEvent' is not assignable to type 'TypedTargetEvent<HTMLInputElement>'.

ERROR in ./src/widgets/WorkerForm.ts
(63,15): error TS2345: Argument of type '{ key: string; placeHolder: string; type: string; onInput: ({ target: { value: email } }: TypedTa...' is not assignable to parameter of type 'Readonly<Partial<TextBox & DeclareCreatedObject> & DijitWrapperProperties> & Readonly<WidgetPrope...'.
  Type '{ key: string; placeHolder: string; type: string; onInput: ({ target: { value: email } }: TypedTa...' is not assignable to type 'Readonly<Partial<TextBox & DeclareCreatedObject> & DijitWrapperProperties>'.
    Types of property 'onInput' are incompatible.
      Type '({ target: { value: email } }: TypedTargetEvent<HTMLInputElement>) => void' is not assignable to type '((e: DocumentEvent) => void) | undefined'.
        Type '({ target: { value: email } }: TypedTargetEvent<HTMLInputElement>) => void' is not assignable to type '(e: DocumentEvent) => void'.
          Types of parameters '__0' and 'e' are incompatible.
            Type 'DocumentEvent' is not assignable to type 'TypedTargetEvent<HTMLInputElement>'.
```

Reply:
Apologies... I changed the final, but forgot to backport that to initial... 😢